.github/workflows/publish.yml

```diff
@@ -6,14 +6,28 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    # Setup .npmrc file to publish to npm
-    - uses: actions/setup-node@v3
-      with:
-        node-version: '12.x'
-        registry-url: 'https://registry.npmjs.org'
-    - run: npm install
-    - run: npm run build
-    - run: npm run publish
-      env:
-        NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+      - name: Build
+        run: |
+          npm ci
+          npm run build
+      - name: Publish
+        run: npm run publish
```

Comment:
could you also create a workflow to publish the website on its own?

Reply:
which should only be triggered manually I guess, because now we can block the main commit without demo code. so website-only-deploys shouldn't happen that often.